markers/sauce.py

```diff
@@ -14,7 +16,22 @@ def pytest_addoption(parser):
 def pytest_configure(config):
     config.addinivalue_line('markers', __doc__.splitlines()[0])
     if config.option.sauce:
+        conf.env['browser']['sauce'] = True
         if config.option.markexpr:
             config.option.markexpr = 'sauce and ({})'.format(config.option.markexpr)
```

Comment:
@psav: we may need to remove this config for not to add marker for us? if we specify --sauce it may end up picking only few tests and not all tests under 'cfme/tests/'